lib/ejson/rails/railtie.rb

```diff
@@ -20,6 +20,13 @@ class Railtie < Rails::Railtie
         Rails.application.credentials.config.deep_merge!(secrets) do |key|
           raise "A credential already exists with the same name: #{key}"
         end
+
+        # Delete the loaded JSON files so they are no longer readable by the app.
+        if ENV["EJSON_RAILS_DELETE_SECRETS"] == "true"
```

Comment:
Using an environment variable is unnecessary. We should've used Rails configuration.